src/scripts/datepickerRange.js

```diff
@@ -0,0 +1,51 @@
+import { DatePicker } from './datepicker';
+
+export class DatePickerRange {
+
+    constructor(options) {
+        this.rightDateSelected = false;
+        this.options = options || {
```

Comment:
The default values are defined in the class DatePicker. Only leftSelector and rightSelector are required.